src/kemal/helpers/macros.cr

```diff
@@ -67,8 +65,10 @@ macro render(filename, layout)
 end
 
 # Render view with the given filename.
-macro render(filename)
-  Kilt.render({{filename}})
+macro render(filename, *args)
+  String.build do |__kemal_io__|
+    ECR.embed({{filename}}, "__kemal_io__", {{*args}})
```

Comment:
ECR.embed has no splat parameters. And there's no need to re-implement ECR.render which already takes care of building a string. This method could just delegate to that.
In fact, I would even suggest to remove Kemal's render macro entirely. If its just an alias to ECR.render, people can just call that directly. Thats also more transparent.

And it allows to properly deprecate Kemal's render macro with a deprecation message, suggesting to use ECR or Kilt directly instead.